app/Actions/User/SaveUserInfoAction.php

```diff
@@ -90,20 +91,24 @@ public function execute(SaveUserInfoRequest $userInfoRequest): SaveUserInfoRespo
         }
 
         if ($userHasNotInfo || !empty($avatar)) {
-            $userInfo->avatar_url = $this->storeAvatar($userInfo->user_id, $avatar);
+            $userInfo->avatar_url = $this->storeAvatar($avatar);
         }
 
         $userInfo = $this->userInfoRepository->save($userInfo);
 
         return new SaveUserInfoResponse($userInfo);
     }
 
-    private function storeAvatar(int $userId, ?UploadedFile $avatar): string
+    private function storeAvatar(?UploadedFile $avatar): string
```

Comment:
if ($userHasNotInfo || !empty($avatar)) {...}

private function storeAvatar(UploadedFile $avatar): string
no need to validate $avatar is null in storeAvatar method